content/docs/references/cli/_index.md

```diff
@@ -782,3 +783,58 @@ Encryption rekey status with name of the rekey object
 # dellctl encryption rekey-status myrekey
  INFO Status of rekey request myrekey = completed
 ```
+
+
+### dellctl images
+
+List the container images needed by csi driver
+
+**NOTE.**: dellctl images Currently supports csi-vxflexos driver only.
```

Comment:
Currently -> currently (first letter caps not needed)

Reply:
done